### PR TITLE
Make it possible to set safety_settings when using the Gemini provider

### PR DESCRIPTION
**Pull request by motin**
This is a more lightweight implementation than [the one recently submitted for the Vertex AI provider](https://github.com/BerriAI/litellm/pull/1190), and it is less dynamic than [the solution for the Palm provider](https://github.com/BerriAI/litellm/blob/d90e04b531692dcb8672909f526c234d0f828595/litellm/llms/palm.py#L119), but it does make it possible to easily specify safety_settings when using the Gemini provider.

### CODE REVIEW

File: litellm/llms/gemini.py [hunk 1]
```diff
@@ -160,7 +163,7 @@ def completion(
         input=prompt,
         api_key="",
         original_response=response,
-        additional_args={"complete_input_dict": {}},
+        additional_args={"complete_input_dict": {}, "safety_settings": safety_settings},
```
Comment (motin) [inline litellm/llms/gemini.py:166]: I don't know the best practices around litellm logging to know if this is a good idea, please advice

Comment (ghost) [reply litellm/llms/gemini.py:166]: that's fine

**Comment by ghost** [comment]
Hi @motin can you share a screenshot of this working? 

If you could share a test (test_completions.py) and update docs for this too (gemini.md) 

that'd be awesome!

**Comment by motin** [comment]
> Hi @motin can you share a screenshot of this working?

Here is a screenshot from a run that uses this PR's branch:
image

> If you could share a test (test_completions.py) and update docs for this too (gemini.md)

I added a test in test_completions.py, but neither palm.md nor vertex.md mentions the safety_settings parameter. Not sure what to add :)

**Comment by Manouchehri** [comment]
I believe this is supported now, I've been using it for awhile.

image

**Pull request closed without merging**